citi-onboarding/DADe-onepage
# feature(model-realizacoes): created realizacoes model, controller and route

## What I did
- Created model, controller and route for realizacoes
- Fixed a problem with realizacoes plural and singular keystone display

## How to test
1. Open the terminal and go to the folder of this project

2. In terminal:
   1. git checkout develop
   2. git pull
   3. git checkout feature/model-realizacoes
   4. git pull origin feature/model-realizacoes
   5. npm install
   6. node index.js

3. In your browser go to http://localhost:3001/admin , and check if the realizacoes list is appearing, then open the "Realizacoes" list. Create a new item (realizacao) and check if there are "Name" and "Date" fields, after the creation check if you can upload a image. 

4. In your browser go to http://localhost:3001/api/realizacoes , and check if the component informations are there showing it's details.
